mlir/tools/mlir-miopen-driver/CMakeLists.txt

```diff
@@ -38,3 +38,7 @@ add_llvm_executable(mlir-miopen-driver
 llvm_update_compile_flags(mlir-miopen-driver)
 target_link_libraries(mlir-miopen-driver PRIVATE ${LIBS})
 mlir_check_link_libraries(mlir-miopen-driver)
+
+# MIOpenDriver.py
+add_custom_target(MIOpenDriver.py ALL
+	${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/../../utils/jenkins/MIOpenDriver.py ${LLVM_BINARY_DIR})
```

Comment:
This still needs a `COMMAND` in front of it, from the looks of other CMake files

Reply:
That is interesting - was it tested? It looked from the documentation that it would be rejected.

Reply:
Yes. I ran it manually and in CI multiple times.

Reply:
Huh

Weird